### ISSUE TITLE
Accessibility: uncategorized

### ISSUE THREAD

**Issue by andreamartz** [andreamartz/mcc-authentic-grace-2.0#30]
1. Need text identifying clothing items in the images grid
2. Need <address> tag around the address info in footer
3. Missing alt text on several images
4. Need to check the color contrast ratio
5. Need blue border around the element that has the focus?
6. Modal needs work: https://www.smashingmagazine.com/2014/09/making-modal-windows-better-for-everyone/
and
set the focus when the modal pops up: https://bitsofco.de/accessible-modal-dialog/
and 
Search for "modal" on https://www.w3.org/TR/wai-aria/
7. Make sure that the hidden "skip-to-content" link is the first thing read aloud by a screen reader.
